src/main/java/com/google/step/finscholar/data/Utils.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar.data;
+
+import javax.servlet.http.HttpServletRequest;
+
+/** The class containing all commonly used helper functions. */
+public class Utils {
+
+  /**
+   * @param request       The current HTTP request being handled.
+   * @param name          The parameter name.
+   * @param defaultValue  The default string if the field does not exist in request.
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client.
+   */
+  public static String getStringParameter(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
Return an Optional value rather than comparing to some arbitrary default value. This is a lot cleaner and less error prone.
https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html

```
public static Optional<String> getStringParameter(.....) {
    return Optional.of(request.getParameter(name));
}
```

Then you can do

```
if (idOfLastItemOptional.isPresent()) {
    // Do something.
}
```

Reply:
Done! Thanks